packages/genesys-spark-components/src/components/stable/gux-time-picker/tests/__snapshots__/gux-time-picker.spec.ts.snap

```diff
@@ -3733,90 +3705,62 @@ exports[`gux-time-picker #render using regional dates should render as expected
     <gux-popup class="gux-time-picker">
       <div class="gux-input-time" slot="target">
         <div class="gux-input-time-container">
-          <input aria-label="Input hours" class="gux-input-time-hours" pattern="^([01]?[0-9]|2[0-3])$" type="text" value="09">
+          <input aria-label="Input hours" class="gux-input-time-hours" pattern="^(0?[1-9]|1[012])$" type="text" value="9">
```

Comment:
Why did this snapshot change? I must be missing something.

Reply:
Unless I'm mistaken, it's expected that this snapshot would change. This is the part of the snapshot test for when the regional dates "toggle" is true. These time pickers are using 12hr instead of 24hr time now, so this snapshot needed to be updated. None of the snapshot tests for the regional dates toggle being false have changed.